municipal-services/property-services/src/main/java/org/egov/pt/producer/Producer.java

```diff
@@ -1,20 +0,0 @@
-package org.egov.pt.producer;
```

Comment:
Why is this file deleted? @hinamakhija-eGov please confirm if it is correct

Reply:
Since enc-client had the same name of Producer file, this file had to be renamed for proper functioning and it was renamed as PropertyProducer. 
Ref: https://github.com/egovernments/DIGIT-Dev/blob/develop/municipal-services/property-services/src/main/java/org/egov/pt/producer/PropertyProducer.java